src/components/standalone/openvpn_tunnel/TunnelManager.vue

```diff
@@ -52,38 +52,44 @@ const props = defineProps<{
 }>()
 
 const { t } = useI18n()
-
 const uciChangesStore = useUciPendingChangesStore()
 
+const RELOAD_INTERVAL = 10000
 const loading = ref(true)
 const tunnels = ref<ServerTunnel[] | ClientTunnel[]>([])
 const selectedTunnel = ref<Tunnel | null>(null)
 const showCreateEditDrawer = ref(false)
 const showDeleteModal = ref(false)
 const showDownloadModal = ref(false)
 const showImportConfigurationDrawer = ref(false)
+const fetchTunnelsIntervalId = ref(0)
 const error = ref({
   notificationTitle: '',
   notificationDescription: '',
   notificationDetails: ''
 })
 
-async function fetchTunnels() {
+async function fetchTunnels(setLoading: boolean = true) {
   try {
-    loading.value = true
+    if (setLoading) {
+      loading.value = true
+    }
     const listTunnelResponse = await ubusCall('ns.ovpntunnel', 'list-tunnels')
     if (props.manageClientTunnels) {
       tunnels.value = listTunnelResponse.data.clients
     } else {
       tunnels.value = listTunnelResponse.data.servers
     }
-    loading.value = false
   } catch (err: any) {
     error.value.notificationTitle = props.manageClientTunnels
       ? t('error.cannot_retrieve_client_tunnels')
       : t('error.cannot_retrieve_server_tunnels')
     error.value.notificationDescription = t(getAxiosErrorMessage(err))
     error.value.notificationDetails = err.toString()
+  } finally {
+    if (setLoading) {
+      loading.value = false
+    }
```

Comment:
You can avoid the `setLoading` status, since you'll be displaying previous valid data anyway

Just use the `loading` variable as loading state (since it's already set to true on initial component load), then  set it to `false` inside the finally block. There's no need to set it to true every time the request is asked because data will be already loaded